functions.php

```diff
@@ -74,3 +74,61 @@ function error_message( $message ) {
 function trailingslashit( $string ) {
 	return rtrim( $string, '/' ) . '/';
 }
+
+/**
+ * Get WordPress version from a directory.
+ *
+ * @param  string $dir Path
+ * @return string
+ */
+function get_wordpress_version( $dir ) {
+	$wpVersion = file_get_contents( trailingslashit( $dir ) . 'src/wp-includes/version.php' );
```

Comment:
That's a good point, this won't get updated on commit. We get the SVN revision, so we should be able to get the changeset from that. Not sure if this will be actually useful.